sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/SchemaUtilTest.java

```diff
@@ -64,15 +64,17 @@ public void testToBeamSchema() throws SQLException {
             JdbcFieldInfo.of("boolean_col", Types.BOOLEAN),
             JdbcFieldInfo.of("char_col", Types.CHAR, 255),
             JdbcFieldInfo.of("date_col", Types.DATE),
-            JdbcFieldInfo.of("decimal_col", Types.DECIMAL),
+            JdbcFieldInfo.of("decimal_col", Types.DECIMAL, Integer.MAX_VALUE),
```

Comment:
Why not keep just `JdbcFieldInfo.of("decimal_col", Types.DECIMAL)` for this?

Reply:
but it means DECIMAL(0,0) zero precision, zero scale. Despite it's possible in Java, it's not possible in real database, I believe. eg. If I attempt to declare such column in Postgres I've got `NUMERIC precision 0 must be between 1 and 1000`. So, I fixed this stub to mimic real DB metadata.

Reply:
If it declares `DECIMAL(0,0)`, won't it uses just default precision and scale in this case?

Reply:
> If it declares `DECIMAL(0,0)`, won't it uses just default precision and scale in this case?

https://github.com/apache/beam/pull/27406#issuecomment-1630313700
From what I read https://www.postgresql.org/docs/current/datatype-numeric.html 0 precision means variable number of digits.